cvmfs/cache_ram.h

```diff
@@ -237,14 +240,11 @@ class RamCacheManager : public CacheManager {
 
  private:
   struct ReadOnlyFd {
-    ReadOnlyFd() : handle(shash::Any()), pos(0), store(NULL) { }
-    ReadOnlyFd(const shash::Any &h, uint64_t pos)
-      : handle(h)
-      , pos(pos)
-      , store(NULL) { }
+    ReadOnlyFd() : handle(kInvalidHandle) { }
```

Comment:
The `is_volatile` and `index` fields should be initialized by the constructors, too